checks.d/openstack.py

```diff
@@ -346,13 +371,31 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         # Mapping of Nova-managed servers to tags
         self.external_host_tags = {}
 
+        # Set proxy settings
+        self.proxies = {
```

Comment:
Ah yes! Definitely, I had forgotten to do this! I realized too when I fixed the `http_check`.